appveyor.yml

```diff
@@ -14,6 +14,7 @@ build:
 test:
   assemblies:
     only:
+    - NHibernate.Caches.CoreMemoryCache.Tests.dll
```

Comment:
Tested as seen in appveyor log, but tests results are not reported in tests tab.